dart/.gitignore

```diff
@@ -0,0 +1,6 @@
+.pub/
```

Comment:
would prefer this to be part of the root `.gitignore`, that way it is easier to overview what is all being ignored